unified-runtime/test/CMakeLists.txt

```diff
@@ -10,15 +10,91 @@ FetchContent_Declare(
   GIT_TAG        release-1.12.1
 )
 
+include(FindLit)
+
 set(UR_TEST_DEVICES_COUNT 1 CACHE STRING "Count of devices on which conformance and adapters tests will be run")
 set(UR_TEST_PLATFORMS_COUNT 1 CACHE STRING "Count of platforms on which conformance and adapters tests will be run")
 set(UR_TEST_FUZZTESTS ON CACHE BOOL "Run fuzz tests if using clang and UR_DPCXX is specified")
 # For Windows: Prevent overriding the parent project's compiler/linker settings
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND UR_DPCXX AND UR_TEST_FUZZTESTS)
+    set(UR_FUZZTESTING_ENABLED ON)
+endif()
+
 set(INSTALL_GTEST OFF)
 FetchContent_MakeAvailable(googletest)
 enable_testing()
 
+# Set up the root `check-unified-runtime` target
+configure_file(lit.site.cfg.py.in lit.site.cfg.py)
+add_custom_target(deps_check-unified-runtime)
+if(UR_STANDALONE_BUILD)
+  add_custom_target(check-unified-runtime
+    COMMAND "${URLIT_LIT_BINARY}" "${CMAKE_CURRENT_BINARY_DIR}" -v
+    DEPENDS deps_check-unified-runtime
+  )
+else()
+  add_lit_testsuite(check-unified-runtime "Running Unified Runtime ${suite} tests"
```

Comment:
Where does `add_lit_testing()` come from?

Reply:
I'm still interested in an answer to this.

Reply:
It's a function defined in intel/llvm. If a full LLVM build is available, we use the lit test suite managing thingies from there.